scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+If not, it is time for GH Projects.
+
+---
+![[projects.png]]
+
+## Projects
+
+notes:
+
+Each GitHub repo, team, and organisation can have a project board, a lightweight kanban board with customisable columns, allowing you to group your issues together, and observe their progress through your current iteration.
+
+This represents the information radiator for your team, a bird's-eye view of what is happening with the project, and something you might gather around for your morning catchup meeting.
+
+You don't need all the features of Trello, JIRA, or anything like that. You need the minimum viable board.
+
+This bare-bones tool completely side-steps 'the JIRA effect', which is if you have a tool that is packed full of time tracking, velocity points, and so on, the temptation is to use all these features, even if they give no value, and complicate your processes.
+
+---
+
+## Aside
+
+# Standups Are Great
+
+notes:
+
+If you have one meeting a day, it should be a standup.
+Good standups replace other meetings and accelerate your project dramatically.
+
+The way I like to do standups is not by asking everyone what they did yesterday and what they intend to do today.
+
+That's a great way to find out at length what Dave did on his day off, but not a good way to find out what's happening with the project specifically.
+
+I recommend walking your board, backwards, from right to left.
+Everyone is timeboxed, keeping the meeting tight.
+If you don't have an issue on the board, you don't speak yet. Perhaps you could write or pick up a task from the backlog and talk about it tomorrow.
+
+At the end, the team member who is facilitating the meeting asks if anyone has any blockers, and we're done.
+
+The most important part of this most important meeting is asking if you have any blockers.
+A good standup means that no-one can be delayed by more than 24 hours.
+
+Note I didn't say the PM or DM or scrum master or whoever facilitates.
+That person doesn't exist in my team.
+These are not roles, they are hats.
+
+I don't know if your team needs any other meetings, but it is vital that you do a standup.
+
+Back to GitHub.
+
+---
+![[org-public.png]]
+
+## Organisations
+
+notes:
+GitHub provides an umbrella group of users called an organisation.
+This is your company, and if you're building your products in the open, as I recommend you do, you won't pay GitHub a thing.
+Most git hosts provide their services for free for open-source companies.
+If you have too much money, you can pay GitHub for a plan to make your data closed.
+
+If you require more subdivision,
+Organisations are divided into Teams.
+
+---
+![[teams.png]]
+
+## Teams
+
+notes:
+
+Teams on GitHub allow you to granularly scope repo, project, wiki, and other permissions to the different teams in your organisation.
+
+I recommend allowing everyone to write and contribute to all projects, you want the network effect and low admin overhead.
+This pattern is called "internal open source".
```

Comment:
To me, "internal open source" is "[innersource](https://about.gitlab.com/topics/version-control/what-is-innersource/)" (google it), which doesn't refer to loose permissions, it refers to trying to developer an open-source-like culture within proprietary software teams.